packages/react/package.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "@stardust-ui/react",
+  "name": "@fluentui/react",
```

Comment:
Teams has a pending question about timeline and OSS approval.  I'm thinking we might setup `@stardust-ui` as a separate package publish for now that just re-exports all the `@fluentui` packages, then once their OSS approval is figured out we can kill that package.

Something like:
```
Teams' OSS Waiver ---------------------------X
                                             |
stardust-ui -----   ...(re-export).......... |
                 \  |                       \|
      fluent-ui   >-^------------------------>---->
```

Reply:
Per offline discussion - I think we can ignore the stardust shim package now.